codesamples/sample.js

```diff
@@ -1,8 +1,8 @@
 async function remindUserWorkflow(userId, intervals) {
```

Comment:
In JS the workflow main function is always called `main`.

```suggestion
// workflows/remind-user.js
async function main(userId, intervals) {
```